### DISCUSSION TITLE
Nitro not upgraded

### DISCUSSION THREAD

**Discussion by zerosdev** [nuxt/nuxt#19540] [Questions] [answered]
I upgraded Nuxt 3.0.0 to the latest with `npx nuxi upgrade` but the nitro still in 1.0.0. How to upgrade ?

------------------------------
- Operating System: `Windows_NT`
- Node Version:     `v16.13.0`
- Nuxt Version:     `3.2.3`
- Nitro Version:    `1.0.0`
- Package Manager:  `npm@8.1.0`
- Builder:          `vite`
- User Config:      `runtimeConfig`, `modules`, `build`, `device`, `auth`
- Runtime Modules:  `@nuxt-alt/auth@2.3.6`, `@pinia/nuxt@0.4.7`, `@nuxtjs/device@3.1.0`, `bootstrap-vue-next/nuxt`
- Build Modules:    `-`
------------------------------

**Comment by AloisSeckar** [comment]
Maybe try with `--force` option, so it also removes and restores node_modules and rebuilds package-lock.json?

**Reply by zerosdev** [reply]
> Maybe try with `--force` option, so it also removes and restores node_modules and rebuilds package-lock.json?

I tried to delete node_modules and package-lock.json but still not upgraded, here is my package.json

```
{
  "private": true,
  "scripts": {
    "build": "nuxt build",
    "dev": "nuxt dev",
    "generate": "nuxt generate",
    "preview": "nuxt preview",
    "postinstall": "nuxt prepare"
  },
  "devDependencies": {
    "@nuxtjs/device": "^3.0.0",
    "@nuxtjs/i18n": "^7.3.1",
    "@popperjs/core": "^2.11.6",
    "@sidebase/nuxt-auth": "^0.3.3",
    "bootstrap-vue-next": "^0.6.2",
    "nuxt": "^3.2.3",
    "sass": "^1.57.1"
  },
  "dependencies": {
    "@nuxt-alt/auth": "^2.1.9",
    "@nuxt-alt/http": "^1.5.7",
    "@pinia/nuxt": "^0.4.7",
    "@vuepic/vue-datepicker": "^4.2.0",
    "awesome-notifications": "^3.1.3",
    "bootstrap": "^5.2.3",
    "dayjs": "^1.11.7",
    "floating-vue": "^2.0.0-beta.20",
    "pinia": "^2.0.28",
    "sweetalert2": "^11.4.8",
    "vue-awesome-paginate": "^1.1.46",
    "vue-currency-input": "^3.0.3",
    "yarn": "^1.22.19"
  }
}
```

**Comment by zerosdev** [comment, marked as answer]
Finally, I found the problem. Nitro can't be upgraded because the @sidebase/nuxt-auth package locks nitro at ^1.0.0 (in my current installed version) which actually I didn't use this package anymore so after I removed it then `npx nuxi upgrade` nitro can be upgraded to the latest
